extra_data/tests/conftest.py

```diff
@@ -108,6 +109,24 @@ def mock_spb_raw_run(format_version):
         yield td
 
 
+@pytest.fixture(scope='session')
+def mock_spb_raw_and_proc_run():
+    with TemporaryDirectory() as td:
+        prop_dir = osp.join(str(td), 'SPB', '201830', 'p002012')
+
+        # Set up raw
+        raw_run_dir = osp.join(prop_dir, 'raw', 'r0238')
+        os.makedirs(raw_run_dir)
+        make_examples.make_spb_run(raw_run_dir)
+
+        # Set up proc
+        proc_run_dir = osp.join(prop_dir, 'proc', 'r0238')
+        os.makedirs(proc_run_dir)
+        make_examples.make_spb_run(proc_run_dir, raw=False)
+
+        yield td, prop_dir, raw_run_dir, proc_run_dir
```

Comment:
I agree that returning all these paths isn't the most elegant. I think you could skip returning `prop_dir` - the only place that it's used, it can easily be replaced with `proc_run_dir`. Then I think it's good enough.

Reply:
Good catch, fixed in f9fef4e.